monkey/infection_monkey/master/exploiter.py

```diff
@@ -142,13 +142,15 @@ def _run_all_exploiters(
             exploiter_name = exploiter.name
             victim_os = target_host.operating_system
 
-            # We want to try all exploiters if the victim's OS is unknown
-            if victim_os is not None and victim_os not in SUPPORTED_OS[exploiter_name]:
-                logger.debug(
-                    f"Skipping {exploiter_name} because it does not support "
-                    f"the victim's OS ({victim_os})"
-                )
-                continue
+            # This is a hack for now
+            if exploiter_name in SUPPORTED_OS:
```

Comment:
I'd say we update `SUPPORTED_OS` with `MockPlugin`. Easier to read, lesser logic to understand when we change this.

Reply:
Yeah, but it's not quite that simple. I'll create an issue to handle it in the new year.